src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -361,6 +369,18 @@ def _monitor_workers_and_process_tasks_in_thread(
                 # Handle process crashed.
                 if crashed:
                     bulk_logger.warning(f"Process crashed while executing line {line_number}.")
+                    logName_i = "{}_{}.log".format(ProcessPoolConstants.PROCESS_LOG_NAME, index)
+                    log_path = ProcessPoolConstants.PROCESS_LOG_PATH / logName_i
+                    # In fork mode, if the child process fails to start, its error information
+                    # will be written to the parent process log file.
+                    # So if 'log_errors_form_path' return 'false', it means the child process fails to start.
+                    # Attempt read the parent process log file.
+                    if not log_errors_from_file(log_path) and self._use_fork:
+                        log_path = (
+                            ProcessPoolConstants.PROCESS_LOG_PATH
+                            / ProcessPoolConstants.SPANED_FORK_PROCESS_MANAGER_LOG_NAME
+                        )
```

Comment:
Better to change the variable name, it's confusing now.
Consider manager_log_path

Reply:
fixed